public/pages/CreateIndex/components/MappingLabel/MappingLabel.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { forwardRef, useCallback, useRef, useImperativeHandle } from "react";
+import {
+  EuiIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiToolTip,
+  EuiCode,
+  EuiBadge,
+  EuiText,
+  EuiContextMenu,
+  EuiFormRowProps,
+} from "@elastic/eui";
+import { set, pick } from "lodash";
+import { MappingsProperties } from "../../../../../models/interfaces";
+import { AllBuiltInComponents } from "../../../../components/FormGenerator";
+import useField, { transformNameToString } from "../../../../lib/field";
+import { INDEX_MAPPING_TYPES, INDEX_MAPPING_TYPES_WITH_CHILDREN } from "../../../../utils/constants";
+import SimplePopover from "../../../../components/SimplePopover";
+
+const OLD_VALUE_DISABLED_REASON = "Old mappings can not be modified";
```

Comment:
Should we use `cannot` instead of `can not`?

Reply:
Agree, fixed on https://github.com/opensearch-project/index-management-dashboards-plugin/commit/99f41b490d9c437390173a0bca9c2e11550035cf.